vagrant/playbooks/user-sandbox.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Unfortunately, there is now 1 new commit on the pulplift master branch that we need.

https://github.com/pulp/pulplift/pull/108/files

(One of them adds a new file in this directory,)

Reply:
Oh yes, that's unfortunate...
I will have a look tomorrow.